apps/core-app/src/contexts/DaoContext.tsx

```diff
@@ -26,6 +26,8 @@ import {
   loadProposalsList,
 } from '../utils/contextHelpers';
 
+export type TDao = DaoWithTokenDataQuery['dao'];
```

Comment:
I'm not psyched on the type names in dao-data - playing with making them more friendly for app dev.